packages/side-nav/src/vaadin-side-nav-helpers.js

```diff
@@ -0,0 +1,38 @@
+import { SideNavItem } from './vaadin-side-nav-item.js';
+
+/**
+ * @param {string} path
+ * @param {SideNavItem} item
+ */
+export function doesPathMatchItem(path, item) {
+  const sanitizedPathToMatch = path.startsWith('/') ? path.substring(1) : path;
+  const sanitizedItemPath = item.path.startsWith('/') ? item.path.substring(1) : item.path;
+  // Try matching the item path
+  if (sanitizedPathToMatch === sanitizedItemPath) {
+    return true;
+  }
+  // Try matching the item path aliases given they exist
+  if (item.pathAliases || item.pathAliases === '') {
+    return item.pathAliases.split(',').some((alias) => {
+      const sanitizedAlias = alias.startsWith('/') ? alias.substring(1) : alias;
+      return sanitizedPathToMatch === sanitizedAlias;
+    });
+  }
+  return false;
+}
+
+/**
+ * @param {Document} document
+ */
+export function getCurrentRelativePath(document) {
```

Comment:
I don't see a need to pass `document` here, maybe we could just global `document` variable.

Reply:
Done.